core-plugins/src/test/java/co/cask/hydrator/plugin/transform/PythonEvaluatorTest.java

```diff
@@ -134,6 +134,30 @@ public void testSimple() throws Exception {
   }
 
   @Test
+  public void testDecodingSimpleTypes() throws Exception {
```

Comment:
This is great. Can we add one more test which corresponds to the exact scenario that failed (refer to the json file attached to the JIRA)?

Python Transform Code:
```
def transform(record, emitter, context):
   output = {}  
   output['body'] = record['body']
   emitter.emit(output)
```